doc/setup/email_gateway.rst

```diff
@@ -0,0 +1,40 @@
+:banner: banners/email_gateway.jpg
+
+==================
+Odoo email gateway
+==================
+
+The Odoo mail gateway is a better alternative than using an "incoming mail server"
+because the messages are received instantaneously.
+
+The principle is straightforward. Your incoming SMTP server will receive emails and directly forward
+them through a script (``odoo/addons/mail/static/scripts/odoo-mailgate.py``).
+
+This script will simply connect to your database through XMLRPC to send the mails to MailThread.message_process().
```

Comment:
The script takes care of connecting to your Odoo database through XML-RPC, and send the emails via the MailThread.message_process() feature.